examples/auth/README.md

```diff
@@ -0,0 +1,29 @@
+# Authentication
+
+<p class="description">An app that shows how to set up authentication in Toolpad.</p>
+
+Example showcasing how to set up authentication providers in a Toolpad app. [The docs](https://mui.com/toolpad/concepts/authentication/).
+
+<a target="_blank">
+  <img src="https://mui.com/static/toolpad/marketing/authentication.png" alt="Authentication" style="aspect-ratio: 131/88;" width="524">
+</a>
+
+## How to run
```

Comment:
Would have to explain here as well how to set up the tokens.
Is it possible to try this example with e.g. only Google? It may make sense to make 2 examples, one for each rpovider.

Reply:
I guess so, but since the providers can work together and not to be redundant I thought I would have them together... Both options could be valid, so not sure, I guess I can split them since it's only 2 providers and they should be pretty popular.

Reply:
I feel like the example loses its purpose if I have to set up both providers to try it out. Too much friction.

Reply:
Got it, I've split the examples and linked from the readmes to the OAuth client / provider specific instructions too.